common/laws/00_distribution_of_power.txt

```diff
@@ -832,6 +900,18 @@ law_universal_suffrage = {
 			limit = { ai_has_enact_weight_modifier_journal_entries = yes }
 			add = 750
 		}
+		if = {
+			limit = { 
+				OR = {
+					has_law = law_type:law_autocracy
+					has_law = law_type:law_technocracy
+					has_law = law_type:law_oligarchy
+					has_law = law_type:law_landed_voting
```

Comment:
The journal entry needs to reflect that. Currently both anarchy and single-party laws are not valid for completion of the journal entry.